<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserRightsEditDialog</class>
 <widget class="QDialog" name="UserRightsEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Профиль прав</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Наименование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="grpFilter">
     <property name="title">
      <string>Фильтр прав</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="lblRightFilter">
        <property name="text">
         <string>&amp;Показывать</string>
        </property>
        <property name="buddy">
         <cstring>cmbActivenessFilter</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbActivenessFilter">
        <item>
         <property name="text">
          <string>Все</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Выбранные</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Не выбранные</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblCodeFilter">
        <property name="text">
         <string>Код</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtCodeFilter"/>
      </item>
      <item>
       <widget class="QLabel" name="lblNameFilter">
        <property name="text">
         <string>Наименование</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtNameFilter"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="CRBCheckTableView" name="tblRights"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Права</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBCheckTableView</class>
   <extends>QTableView</extends>
   <header>library/RBCheckTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtName</tabstop>
  <tabstop>cmbActivenessFilter</tabstop>
  <tabstop>edtCodeFilter</tabstop>
  <tabstop>edtNameFilter</tabstop>
  <tabstop>tblRights</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserRightsEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserRightsEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
